geode-core/src/main/java/org/apache/geode/management/internal/cli/CliUtil.java

```diff
@@ -127,7 +128,14 @@ public static InternalCache getCacheIfExists() {
     return cache;
   }
 
-  public static byte[][] filesToBytes(String[] fileNames) throws IOException {
+  /**
+   * even thought this is only used in a test, caller of MemberMXBean.processCommand(String, Map,
+   * Byte[][]) will need to use this method to convert a fileList to Byte[][] to call that
+   * deprecated API.
+   *
+   * Once that deprecated API is removed, we can delete this method and the tests
+   */
```

Comment:
Please use proper sentences in javadocs - Capitalize 'Even', add closing period.